job-utilities/job-router/requirements.txt

```diff
@@ -0,0 +1,3 @@
+boto3
+urllib3 >= 2
+jmespath
```

Comment:
What purpose does this file serve? Is this a python library management thing?

Reply:
Yeah pretty much. You can run pip (python library installing tool) with the file as input